### DISCUSSION TITLE
Case insensitive order by?

### DISCUSSION THREAD

**Discussion by bpuig** [CodeDredd/pinia-orm#634] [Q&A] [answered]
When I use orderBy, first I get the Uppercase and then the lowercase results. Is it possible to orderBy without casing?
Thanks in advance

**Comment by CodeDredd** [comment, marked as answer]
Right now its not possible. But if its ok i am going to include this feature to the `sortBy` helper because `orderBy` can by used multiple times with different fields where setting a sorting flag would be confusing.

So i try to implemnt something like this:
````ts
const userCollection = useCollect(useRepo(User).make([
    { id: 1, name: 'James', age: 40, post: { id: 1, title: 'Title1' } },
    { id: 2, name: 'john', age: 30, post: { id: 2, title: 'Title2' } },
    { id: 3, name: 'David', age: 20 },
  ])).sortBy('name', 'SORT_FLAG_CASE')

````

**Reply by bpuig** [reply]
Thank you very much!
